qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/completions/QuteCompletionsForExpression.java

```diff
@@ -715,12 +715,13 @@ public void doCompleteNamespaceResolvers(String namespace, Template template, Ra
 		}
 	}
 
-	private void doCompleteExpressionForObjectPartWithParentNodes(Node part, Node node, Range range, int offset,
-			Template template, Set<String> existingVars, QuteCompletionSettings completionSettings,
-			QuteFormattingSettings formattingSettings, QuteNativeSettings nativeImagesSettings, CompletionList list) {
+	private CompletableFuture<CompletionList> doCompleteExpressionForObjectPartWithParentNodes(Node part, Node node,

```

Comment:
Please don't update this signature and do like #with in https://github.com/redhat-developer/quarkus-ls/blob/37bb5f95f533d72fb09c817a85ac634c3684adf5/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/completions/QuteCompletionsForExpression.java#L820